packages/@aws-cdk/aws-scheduler-alpha/test/schedule.test.ts

```diff
@@ -0,0 +1,65 @@
+import { App, Stack } from 'aws-cdk-lib';
+
+import { Template } from 'aws-cdk-lib/assertions';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { IScheduleTarget, Schedule, ScheduleTargetConfig } from '../lib';
+import { ScheduleExpression } from '../lib/schedule-expression';
+
+class SomeLambdaTarget implements IScheduleTarget {
+  public constructor(private readonly fn: lambda.IFunction, private readonly role: iam.IRole) {
+  }
+
+  public bind(): ScheduleTargetConfig {
+    return {
+      arn: this.fn.functionArn,
+      role: this.role,
+    };
+  }
+}
+
+describe('Schedule', () => {
+  let stack: Stack;
+  let func: lambda.IFunction;
+  const expr = ScheduleExpression.at(new Date(Date.UTC(1969, 10, 20, 0, 0, 0)));
+  let role: iam.IRole;
```

Comment:
```suggestion
  let role: iam.IRole;
  const expr = ScheduleExpression.at(new Date(Date.UTC(1969, 10, 20, 0, 0, 0)));
```
`role` declaration should be moved together with other `let` variables